storm-client/src/jvm/org/apache/storm/blobstore/BlobStoreFile.java

```diff
@@ -42,6 +42,10 @@ public abstract class BlobStoreFile {
 
     public abstract long getModTime() throws IOException;
 
+    public long getVersion() throws IOException {
```

Comment:
I didn't make it abstract because we would need to implement it on HdfsBlobStoreFile with this default implementation we can maintain backward compatibility